tests/geometric/geometric_2d.cpp

```diff
@@ -0,0 +1,123 @@
+#include <gtest/gtest.h>
+
+#include <cmath>
+
+#include "kagen/context.h"
+#include "kagen/definitions.h"
+#include "kagen/generators/geometric/geometric_2d.h"
+#include "kagen/generators/geometric/rgg.h"
+#include "tests/util/utils.h"
+
+using namespace kagen;
+
+TEST(GEOMETRIC, generates_graph_on_one_PE) {
+    PGeneratorConfig config;
+    config.n           = 32;
+    config.r           = 0.5;
+    config.coordinates = true;
+
+    RGG2DFactory factory;
+    config         = factory.NormalizeParameters(config, 0, 1, false);
```

Comment:
Tests that only run on one PE should be moved to their own binary / cc file, so that they can be registered with the right number of cores in the CMakeLists